src/plugins/data/public/search/search_interceptor.ts

```diff
@@ -183,6 +184,11 @@ export class SearchInterceptor {
     request: IKibanaSearchRequest,
     options: ISearchOptions = {}
   ): Observable<IKibanaSearchResponse> {
+    options = {
+      strategy: ES_SEARCH_STRATEGY,
```

Comment:
search_interceptor will be simplified in follow-up pr. for now I had to specify non async search strategy here, otherwise it would hit async and won't handle partial results.